bitnami/pytorch/values.yaml

```diff
@@ -0,0 +1,191 @@
+## Global Docker image parameters
```

Comment:
This is the default values.yaml file. I would disable distributed mode in this one and create a values-production.yaml  with distributed mode enabled